build.gradle

```diff
@@ -7,6 +7,7 @@ buildscript {
     dependencies {
         classpath 'org.github.ngbinh.scalastyle:gradle-scalastyle-plugin_2.11:1.0.1'
         classpath 'com.commercehub.gradle.plugin:gradle-avro-plugin:0.16.0'
+        classpath 'com.adtran:scala-multiversion-plugin:1.+'
```

Comment:
Initially I wanted to go with cross-compilation for 2.11/2.12, but now that we're upgrading to Spark 3 too we'll only build for 2.12.

This cross-compilation plugin seems to work well, though, so keeping it around for future 2.13 support.